src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +123,21 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Returns sorted list of people
+     * @return sortedPersonList
+     */
+    public ArrayList<ReadOnlyPerson> sort() {
+        ArrayList<ReadOnlyPerson> sortedPersonList = new ArrayList<>(internalList);
+        sortedPersonList.sort(new Comparator<ReadOnlyPerson>() {
+            @Override
+            public int compare(ReadOnlyPerson o1, ReadOnlyPerson o2) {
+                return o1.getName().toString().compareTo(o2.getName().toString());
```

Comment:
Ideally, you should break before having another dot (.) operator as per the coding standard.